src/components/Books/Books.jsx

```diff
@@ -1,11 +1,16 @@
-import { useSelector } from 'react-redux';
-import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import React, { useEffect } from 'react';
+import { BallTriangle } from 'react-loader-spinner';
 import Book from './Book';
 import Form from './Form';
+import { getBooks } from '../../redux/books/bookSlice';
 
 const Books = () => {
-  const { bookstore: { bookStore: books } } = useSelector((store) => store);
-  // console.log(books);
+  const { bookstore: { bookStore: books, booksLoading: loading } } = useSelector((store) => store);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
```

Comment:
# thanks For the information 

hi @Qoosim  I hope you are doing well

I can't see your video, however may be i understand what you are saying let me check this  

thanks for your time, it would be better if you provide me the video of debugging on my slack(Adarsh Pathak) or  [mail me](mailto:adi8090808766@gmail.com)